tests/foreman/api/test_multiple_paths.py

```diff
@@ -474,6 +475,10 @@ def test_delete_and_get(self, entity):
             self.fail(err)
         logger.info('test_delete_and_get path: {0}'.format(entity_n.path()))
         entity_n.delete()
+        if entity == entities.Repository:
```

Comment:
OK. I'll do this. We'll probably have to deal with this again in the future. But there's a possibility that this bug fix will make its way back to 6.0.x, so it does make sense to re-visit this again.